lesson_py1/t2.py

```diff
@@ -0,0 +1,11 @@
+# Найдите сумму цифр трехзначного числа.
+
+inp = input("Input three digit number: ")
+
+if inp.isdigit():
+    if len(inp) == 3:
```

Comment:
Эти два условия можно легко совместить
```suggestion
if inp.isdigit() and len(inp) == 3:
```